book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] [optional] Kindly consider changing your add rental method to allow the use of two parameters person and date, also while creating the rental the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.

Reply:
- [ ] [optional} Kindly consider adding a method for adding rentals in your person class, which will be similar to the one in the book class